app/global-components.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+
```

Comment:
I think It would be better to create folder `components` and there in different files to store each component. It would improve searching for correct files in the future.